packages/pieces/community/mollie/src/lib/actions/create-order.ts

```diff
@@ -0,0 +1,75 @@
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { mollieAuth } from '../common/common';
+import { MollieApi } from '../common/common';
+
+export const createOrderAction = createAction({
+  auth: mollieAuth,
+  name: 'create_order',
+  displayName: 'Create Order',
+  description: 'Create a new order in Mollie',
+  props: {
+    amount: Property.Number({
+      displayName: 'Total Amount',
+      description: 'Total order amount',
+      required: true,
+    }),
+    currency: Property.StaticDropdown({
+      displayName: 'Currency',
+      required: true,
+      defaultValue: 'EUR',
+      options: {
+        options: [
+          { label: 'EUR', value: 'EUR' },
+          { label: 'USD', value: 'USD' },
+          { label: 'GBP', value: 'GBP' },
+        ],
+      },
+    }),
```

Comment:
Replace Number + separate currency with Mollie’s required decimal string format and clearer UX.
```suggestion
    amount_currency: Property.StaticDropdown({
  displayName: 'Amount Currency',
  required: true,
  defaultValue: 'EUR',
  options: { options: [{ label: 'EUR', value: 'EUR' }, { label: 'USD', value: 'USD' }, { label: 'GBP', value: 'GBP' }] },
}),
amount_value: Property.ShortText({
  displayName: 'Amount Value',
  description: 'Exact amount string with 2 decimals (e.g. 1000.00)',
  required: true,
}),
```